larky/src/main/java/com/verygood/security/larky/ModuleSupplier.java

```diff
@@ -43,22 +43,28 @@
 import com.verygood.security.larky.modules.globals.PythonBuiltins;
 import com.verygood.security.larky.modules.testing.AssertionsModule;
 import com.verygood.security.larky.modules.testing.UnittestModule;
-import com.verygood.security.larky.modules.types.LarkyType;
+//import com.verygood.security.larky.modules.types.LarkyBuiltinObject;
+//import com.verygood.security.larky.modules.types.LarkyType;
 
 import net.starlark.java.annot.StarlarkBuiltin;
 import net.starlark.java.eval.StarlarkValue;
 
 /**
  * A supplier of modules for Larky
- */
+*/
 public class ModuleSupplier {
 
   public static final ImmutableSet<Class<?>> CORE_MODULES = ImmutableSet.of(
     LarkyGlobals.class,
-    PythonBuiltins.class,
-    LarkyType.class
+    PythonBuiltins.class
+  );
+
+  public static final ImmutableMap<String, Object> CORE_ENVIRONMENT = ImmutableMap.of(
+//    "object", LarkyPyObject.getInstance(),
+//    "type", LarkyTypeClass.getInstance()
```

Comment:
Why `CORE` is commented and being mentioned further

Reply:
This is just temporary until the next few PRs that *do* use CORE environment. It will just be an empty dict for now, so it is basically status quo (the same thing we do currently). That's why this PR exists, so that we are aware that we can now use this `CORE_ENVIRONMENT` to setup values that are to be exposed to the larky runtime environment.